coachmark/src/main/java/com/mercadolibre/android/andesui/coachmark/view/walkthroughmessage/WalkthroughMessageView.kt

```diff
@@ -18,18 +23,50 @@ class WalkthroughMessageView @JvmOverloads constructor(
 ) : ConstraintLayout(context, attrs, defStyleAttr) {
 
     private var position: WalkthroughMessagePosition = WalkthroughMessagePosition.BELOW
+    private var listener: WalkthroughScrollessMessageView.WalkthroughButtonClicklistener? = null
 
     init {
         inflate(context, R.layout.andes_walkthrough_message, this)
         walkthroughTitle.typeface = context.getFontOrDefault(R.font.andes_font_semibold)
         walkthroughDescription.typeface = context.getFontOrDefault(R.font.andes_font_regular)
+        walkthroughNextButton.typeface = context.getFontOrDefault(R.font.andes_font_semibold)
     }
 
-    fun setData(data: WalkthroughMessageModel) {
-        walkthroughTitle.text = data.title
-        walkthroughDescription.text = data.description
-        walkthroughTitle.visibility = View.VISIBLE
-        walkthroughDescription.visibility = View.VISIBLE
+    fun setListener(l: WalkthroughScrollessMessageView.WalkthroughButtonClicklistener) {
```

Comment:
Don't use generic names for variables
```suggestion
    fun setListener(walkthroughButtonClicklistener: WalkthroughScrollessMessageView.WalkthroughButtonClicklistener) {
```